vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -194,85 +226,92 @@ func (d *VolumeDriver) UnmountVolume(name string) error {
 // (until Mount is called).
 // Name and driver specific options passed through to the ESX host
 
-// Create - create a volume.
+// Create submits a volume creation request.
 func (d *VolumeDriver) Create(r volume.Request) volume.Response {
 
 	if r.Options == nil {
 		r.Options = make(map[string]string)
 	}
+
 	// If cloning a existent volume, create and return
-	if _, result := r.Options["clone-from"]; result == true {
-		errClone := d.ops.Create(r.Name, r.Options)
-		if errClone != nil {
-			log.WithFields(log.Fields{"name": r.Name, "error": errClone}).Error("Clone volume failed ")
-			return volume.Response{Err: errClone.Error()}
+	if _, ok := r.Options["clone-from"]; ok {
+		if err := d.ops.Create(r.Name, r.Options); err != nil {
+			log.WithField("name", r.Name).WithError(err).Error(
+				"Clone volume failed")
```

Comment:
Hi @govint,

Which white spaces? Any white space you see there is due to [`gofmt`](https://golang.org/cmd/gofmt/) and should remain there if your team and this project adheres to it. I have not yet encountered a Go project that does not since `gofmt` is the official syntax tool for Go.